pkg/vgmanager/vgmanager_controller.go

```diff
@@ -373,7 +372,11 @@ func (r *VGReconciler) addDevicesToVG(vgName string, devices []internal.BlockDev
 
 	args := []string{vgName}
 	for _, device := range devices {
-		args = append(args, fmt.Sprintf("/dev/%s", device.KName))
+		if device.DiskByPath != "" {
+			args = append(args, device.DiskByPath)
+		} else {
+			args = append(args, device.KName)
```

Comment:
don't think it will work if complete device name with path (for example: `dev/sdb`)  is not provided in the args

Reply:
It should work as we are getting the full path now.